src/interface/meta.thrift

```diff
@@ -1123,8 +1123,9 @@ struct VerifyClientVersionResp {
 
 
 struct VerifyClientVersionReq {
-    1: required binary version = common.version;
+    1: required binary client_version = common.version;
     2: common.HostAddr host;
+    3: binary build_version;
```

Comment:
why need build_version?
if the thrift versions are in the same, client could connect with server too.

Reply:
Not For verify，Just for `show hosts` to show version